src/System.Net.Security/tests/UnitTests/System/Security/Authentication/AuthenticationExceptionTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Xunit;
+
+namespace System.Security.Authentication
+{
+    public class AuthenticationExceptionTest
+    {
+        [Fact]
+        public void Constructor()
+        {
+            AuthenticationException authenticationException = new AuthenticationException();
+
+            Assert.Equal("System error.", authenticationException.Message);
+        }
+
+        [Fact]
+        public void Constructor_String()
```

Comment:
The tests follow a naming pattern like, featureundertest_actionbeingdone_expectedresult. Refer to other tests in this project..